fn/src/Web/Fn.hs

```diff
@@ -202,11 +202,18 @@ fallthrough a ft =
 -- NOTE(dbp 2015-11-05): This list taken from snap-core, BSD3 licensed.
```

Comment:
technically this comment is not as true anymore

Reply:
Good point, could you just add a second comment beneath this one with your initials making it clear that you've expanded on the original list?